source/macos/AdaptiveCards/AdaptiveCards/Views/ACRButton.swift

```diff
@@ -44,7 +44,7 @@ class ACRButton: FlatButton {
         setupButtonStyle()
     }
     
-    init(frame: NSRect = .zero, wantsChevron: Bool = false, wantsIcon: Bool = false, iconNamed: String = "", iconImageFileType: String = "", iconPosition: NSControl.ImagePosition = .imageLeft, style: ActionStyle = .default) {
+    init(frame: NSRect = .zero, wantsChevron: Bool = false, wantsIcon: Bool = true, iconNamed: String = "", iconImageFileType: String = "", iconPosition: NSControl.ImagePosition = .imageLeft, style: ActionStyle = .default) {
```

Comment:
Guess this should only be `true` if the button has `iconURL`?

Reply:
This is temporary. For testing purpose.

Reply:
Removed it now.